src/main/resources/templates/visasearch.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
```

Comment:
What does this page do? Not understanding 'visa search' - unless this finds my missing credit card! :p